src/SharedPipelineComponents/Datalock/SFA.DAS.CollectionEarnings.DataLock.UnitTests/Tests/ScenarioTests/datalock.feature

```diff
@@ -15,3 +15,36 @@ Scenario: The commitment has both a DLOCK_09 and a DLOCK_07 -
 
 	Then I get 1 validation errors in the DataLockValidationResult 
 	And The DatalockValidatioResult contains DLOCK_09
```

Comment:
Quick question. I understood that we weren't doing teh suppressing other locks when DLOCK09 occurs, but this seems to test this by saying i get 1 validation error